app/src/androidTest/java/com/github/sdp/ratemyepfl/database/CourseRepositoryTest.kt

```diff
@@ -0,0 +1,97 @@
+package com.github.sdp.ratemyepfl.database
+
+import com.github.sdp.ratemyepfl.model.items.Course
+import com.google.firebase.firestore.DocumentSnapshot
+import com.google.firebase.firestore.QueryDocumentSnapshot
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.Mockito.mock
+import org.mockito.junit.MockitoJUnitRunner
+import javax.inject.Inject
+
+@HiltAndroidTest
+@ExperimentalCoroutinesApi
+class CourseRepositoryTest {
+
+    @get:Rule
+    var hiltRule = HiltAndroidRule(this)
+
+
+    @Inject
+    lateinit var repository: ItemRepository<Course>
+
+    @Before
+    fun setup() {
+        hiltRule.inject()
+    }
+
+    @ExperimentalCoroutinesApi
+    @Test
+    fun getItemsReturnsCorrectItems() = runTest {
```

Comment:
Are these tests useful ? They are just testing the fake repository.

Reply:
No, they are not. I wanted to use mockito there too but my attempt was unsuccessful. I will remove them.